cmd/snapctl/flags.go

```diff
@@ -129,7 +125,7 @@ var (
 	flTaskMaxFailures = cli.IntFlag{
 		Name:  "max-failures",
 		Usage: "The number of consecutive failures before snap disable the task (Default 10 consective failures)",
-		Value: DefaultMaxFailures,
+		Value: -1,
```

Comment:
We should probably document this somewhere, @IRCody, but from what I understand using a value of `-1` is quite common if you want the *default value* for something to be used. In this case, the default value is `10`, and the default value (`-1`) in `snapctl` will ensure that the underlying default value from the scheduler package is used (which happens to be `10`).